packages/api/src/models/medical/cq-directory.ts

```diff
@@ -8,23 +8,22 @@ export class CQDirectoryEntryModel
   implements CQDirectoryEntry
 {
   static NAME = "cq_directory_entry";
-  declare oid: string;
+  declare id: string;
```

Comment:
I think it's worth adding a comment here and on the interface indicating this is the CQ org's OID, not a UUID like our regular models.